src/index.js

```diff
@@ -1 +1,63 @@
 import './style.css';
+
+const scoreList = document.getElementById('scorelist');
+const userName = document.getElementById('user');
+const scoreMark = document.getElementById('score');
+const submitButton = document.getElementById('submitButton');
+const refreshButton = document.getElementById('refreshButton');
+
+let scores = [];
+const apikey = 'jkEzbR6fNifTU9BFqJSn';
+async function displayList() {
```

Comment:
- [ ] It looks like you're using ES5 in place to arrow functions, convert this in arrow function and you'll be ready for the next project. Good luck!
- Following the arrow function structure will help to be more according to ES6 best practices.